lib/wit/deps/fastly/compute.wit

```diff
@@ -136,10 +136,10 @@ interface uap {
   use types.{error};
 
   resource user-agent {
-    family: func(max-len: u64) -> string;
-    major: func(max-len: u64) -> string;
-    minor: func(max-len: u64) -> string;
-    patch: func(max-len: u64) -> string;
+    family: func(max-len: u64) -> result<string, error>;
+    major: func(max-len: u64) -> result<string, error>;
+    minor: func(max-len: u64) -> result<string, error>;
+    patch: func(max-len: u64) -> result<string, error>;
```

Comment:
This bit seems unrelated; is this an intentional inclusion in this PR?

Reply:
Yes. This was an omission from the original pr: those functions pass in a max-length for the resulting string, but had no way to signal an error. Given that I was making these changes, I updated this at the same time.